ui-grid/ui-grid.d.ts

```diff
@@ -3767,7 +3767,7 @@ declare module uiGrid {
          */
         sortCellFiltered?: boolean;
         /** Algorithm to use for sorting this column */
-        sortingAlgorithm?: (a: any, b: any) => number;
+        sortingAlgorithm?: (a: any, b: any, rowA: any, rowB: any, direction: any) => number;
```

Comment:
a and b shouldn't be `TEntity`? maybe direction set as `uiGridConstants.DESC | uiGridConstants.ASC`